great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -98,7 +109,7 @@ def get_config_with_variables_substituted(
         be optional in GE Cloud mode).
         """
         if not config:
-            config = self.config
+            config = self._project_config
```

Comment:
Same question here, why not use the public self.config accessor?

Reply:
done!